packages/react-components/react-textarea/src/stories/Textarea/TextareaHeight.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { makeStyles, tokens, useId, Label, Textarea } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    '& > label': {
+      marginBottom: tokens.spacingVerticalMNudge,
+    },
+  },
+  textarea: {
+    height: '200px',
+  },
+});
+
+export const Height = () => {
+  const textareaId = useId('textarea');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.base}>
+      <Label htmlFor={textareaId} style={{ display: 'block' }}>
```

Comment:
Could you move this `inline` style into a classname in the `useStyles` call above? It would be better for our examples to show this better practice.